src/main/resources/application.yaml

```diff
@@ -1,6 +1,6 @@
 spring:
   data:
     mongodb:
-      host: localhost
+      host: mymongo
```

Comment:
didn't get this? why we change this?

Reply:
We are using Docker to spin up a mongo container. Refer to the `docker-compose.yml` file. The hostname of the mongo container is listed there.
But I see your point. These can be configured as environment variables and used appropriately to minimize confusion and dependency.